params/hooks_libevm.go

```diff
@@ -141,6 +148,16 @@ func (r RulesExtra) PrecompileOverride(addr common.Address) (libevm.PrecompiledC
 	if p, ok := r.precompileOverrideBuiltin(addr); ok {
 		return p, true
 	}
+
+	// Handle p256verify as a stateless precompile
+	if addr == common.HexToAddress("0x0000000000000000000000000000000000000100") {
```

Comment:
same comment as above: 
* we should not define the address for the precompile
* we should follow a similar approach  to register these as in libevm/go-ethereum


probably best to introduce a new helper function to register these like we do with `precompileOverrideBuiltin`.